cmd/version.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+var (
+	versionCmd = &cobra.Command{
+		Use:   "version",
+		Short: "Print the version number of HWSC service and ElasticSearch",
+		Long:  "Run this command to display version number",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println("HWSC version: v0.0.0")
+			res, err := es7Client.Info()
+			if err != nil {
+				fmt.Println(err)
+			}
+			fmt.Println("ElasticSearch version: ", res)
+		},
```

Comment:
Can you post a screenshot of the result when running the command `version`?

Reply:
![version](https://user-images.githubusercontent.com/35046106/59071042-eb303d80-8871-11e9-8d55-c39994e8303a.PNG)